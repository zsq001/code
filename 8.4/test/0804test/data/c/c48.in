24
10 9 11 20 18 19 1 1 1 1 2 2 4 2 3 1 1 1 12 6 4 10 30 50
11
2
3
4
23
4
18
3
9
2
3
14
