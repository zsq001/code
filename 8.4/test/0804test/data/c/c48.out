7
5
4
1
4
1
5
1
7
5
1
